Makefile

```diff
@@ -598,6 +598,11 @@ prune_deps: clone_deps
 		cd ${DEPS_DIR}/$$dep && $(MAKE) prune; \
 	done
 
+stop_deps:
+	@for dep in ${DEPS} ; do \
```

Comment:
I have the docker compose stop fallback, until we have stop target in each dependency. (but in reality, for openfoodfacts-auth it does not work, because we need the `RUN` version of docker compose...)